samples/js/js-customer-registry/package-lock.json

```diff
@@ -20,24 +20,24 @@
       }
     },
     "@grpc/proto-loader": {
-      "version": "0.6.13",
-      "resolved": "https://registry.npmjs.org/@grpc/proto-loader/-/proto-loader-0.6.13.tgz",
-      "integrity": "sha512-FjxPYDRTn6Ec3V0arm1FtSpmP6V50wuph2yILpyvTKzjc76oDdoihXqM1DzOW5ubvCC8GivfCnNtfaRE8myJ7g==",
+      "version": "0.6.12",
+      "resolved": "https://registry.npmjs.org/@grpc/proto-loader/-/proto-loader-0.6.12.tgz",
+      "integrity": "sha512-filTVbETFnxb9CyRX98zN18ilChTuf/C5scZ2xyaOTp0EHGq0/ufX8rjqXUcSb1Gpv7eZq4M2jDvbh9BogKnrg==",
       "requires": {
         "@types/long": "^4.0.1",
         "lodash.camelcase": "^4.3.0",
         "long": "^4.0.0",
-        "protobufjs": "^6.11.3",
+        "protobufjs": "^6.10.0",
         "yargs": "^16.2.0"
       }
     },
     "@kalix-io/kalix-javascript-sdk": {
-      "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/@kalix-io/kalix-javascript-sdk/-/kalix-javascript-sdk-1.0.0.tgz",
-      "integrity": "sha512-ghRd4AenFuL+XprRzgQpPb4RULVJoSHcE54xQYKQox6Us3gUNH1vzKQ3jVCFUMG4y9qRZmJCfqsterANcyO8Gg==",
+      "version": "1.0.2",
+      "resolved": "https://registry.npmjs.org/@kalix-io/kalix-javascript-sdk/-/kalix-javascript-sdk-1.0.2.tgz",
+      "integrity": "sha512-xf7FhSyTFw5/4jHfuyppx4fPR1H7v6FS31wmqVDziVB3suLa5jxxsJrY/vmwqKu39k6/S5cnH2TsxfuQ8UOJeg==",
       "requires": {
-        "@grpc/grpc-js": "^1.6.7",
-        "@grpc/proto-loader": "^0.6.12",
+        "@grpc/grpc-js": "1.6.7",
+        "@grpc/proto-loader": "0.6.12",
```

Comment:
This is the fixed versions for grpc-js and proto-loader now. Looks like it could be proto-loader 0.6.13, with protobufjs 6.11.3 instead though.